ci/jenkins/Jenkinsfile_py3-master_gpu_doc

```diff
@@ -0,0 +1,51 @@
+// -*- mode: groovy -*-
+
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// Jenkins pipeline
+// See documents at https://jenkins.io/doc/book/pipeline/jenkinsfile/
+
+// timeout in minutes
+max_time = 120
+
+node {
+  // Loading the utilities requires a node context unfortunately
+  checkout scm
+  utils = load('ci/jenkins/utils.groovy')
+  build_steps = load('ci/jenkins/build_steps.groovy')
+}
+utils.assign_node_labels(linux_gpu: 'linux-gpu', linux_cpu: 'linux-cpu')
+
+utils.main_wrapper(
+core_logic: {
+  utils.parallel_stage('Doc Test', [
+    build_steps.test_doctest('gluon-nlp-cpu-py3', 'cpu/py3',
```

Comment:
this is not using the master branch yet as there's ongoing issue with master branch.